versioned_docs/version-1.3.0/use-cases/distributed-mongodb-for-multicloud.mdx

```diff
@@ -0,0 +1,1069 @@
+
+17. On the **Create API Key** page, under **Save API Key Information**, copy the public and private keys for later use.
+
+   ![mongodb](/images/version1.3.0/use-cases/configure-mongodb/save-api-key-info.png)
+
+18. Switch to the master cluster and add the public and private keys that you copied from the Ops Manager DB to the Kubernetes 
+    secret using the following command:
+
+
+    ```
+    kubectl apply -f - <<EOF
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: multi-organization-secret
+      namespace: mongodb
+    stringData:
+      publicKey: mnakubte
+      privateKey: a9ac2905-d28a-4114-ba46-02bf0d904ba3
+    EOF
+    ```
+
+19 You need Organizations ID to add to the Kubernetes secret. To get the ID:
+
+   1. Go to the Ops Manager and go to **Organizations** on the left sidebar.
+   2. Click **ops-manager-db**.
+   3. Go to **Settings** under **Organization** on the left sidebar.
+   4. On the **Organization Settings** page, under **General Settings**, copy the **Organization ID** and save it for later use.
+
+   ![mongodb](/images/version1.3.0/use-cases/configure-mongodb/organization-settings.png)
+
+20. On the master cluster, add the Organization ID that you copied and saved in the following command and apply it.
+
+    ```
+    kubectl apply -f - <<EOF
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: multi-project
+      namespace: mongodb
+    data:
+      baseUrl: http://34.23.212.14:8080
+      orgId: 640f0bf457082e60d2620022
+    EOF
+    ```
+
+### Step 8: Create a MongoDB ReplicaSet using MongoDBMultiCRD
+
+:::caution
+Make sure the following environment variables are set for `MDB_1`, `MDB_2`, and MDB_3 in your shell as per your MongoDB member clusters.
+
+Example:
+
+```
+echo $MDB_1 $MDB_2 $MDB_3
+k8s-cluster-1 k8s-cluster-2 k8s-cluster-3
+```
+:::
+
+1. Apply the following command on the master cluster after setting the environment variables using the following command:
+
+
+```
+$export MDB_VERSION=6.0.2-ent
+```
+```
+kubectl apply -f - <<EOF
+apiVersion: mongodb.com/v1
+kind: MongoDBMulti
+metadata:
+   name: multi-replica-set
+   namespace: mongodb
+spec:
+   version: "${MDB_VERSION}" 
+   type: ReplicaSet
+   persistent: true 
+   duplicateServiceObjects: true 
+   credentials: multi-organization-secret 
+   opsManager:
+     configMapRef:
+       name: multi-project 
+   clusterSpecList:
+     clusterSpecs:
+     - clusterName: ${MDB_1} 
+       members: 1
+     - clusterName: ${MDB_2}
+       members: 1
+     - clusterName: ${MDB_3}    
+       members: 1
+EOF
+```
+
+2. (Optional) If the replicas are not created, then verify the operator logs to see any connection error.
+
+   Use the following command to view the operator logs:
+   ```
+   POD=$(kubectl -n mongodb get po|grep operator|awk '{ print $1 }')
+   ```
+
+   ```
+   kubectl -n mongodb logs -f po/$POD -c mongodb-enterprise-operator |grep IP_ADDRESS_NOT_ON_ACCESS_LIST
+   ```
+
+   Example Output
+
+  ```yaml
```

Comment:
Indentation of the entire output should start below Example. Check the HTML version